tests/test_logsource.py

```diff
@@ -159,7 +159,7 @@ def check_service(product,service,fieldname) -> bool:
 
 
         self.assertEqual(files_with_fieldname_issues, [], Fore.RED +
-                         "There are rule files which contains unkown field or with cast error")        
+                         "There are rule files which contains unkown field or with cast error, to fix FP update <logsource.json>")
```

Comment:
This is a confusing message for contributors. Needs to be more generic and don't mention internal files to fix.